sw/device/silicon_creator/manuf/skus/earlgrey_a0/generic/BUILD

```diff
@@ -168,13 +168,13 @@ opentitan_ram_binary(
         "//sw/device/lib/testing:otp_ctrl_testutils",
         "//sw/device/lib/testing:pinmux_testutils",
         "//sw/device/lib/testing/json:provisioning_command",
-        "//sw/device/lib/testing/json:provisioning_data",
         "//sw/device/lib/testing/test_framework:check",
         "//sw/device/lib/testing/test_framework:ottf_console",
         "//sw/device/lib/testing/test_framework:ottf_test_config",
         "//sw/device/lib/testing/test_framework:status",
         "//sw/device/lib/testing/test_framework:ujson_ottf",
         "//sw/device/silicon_creator/manuf/lib:flash_info_fields",
+        "//sw/device/silicon_creator/manuf/lib:individualize_sw_cfg_earlgrey_a0_sku_test",
```

Comment:
Oops, breaks the build. This got changed in an earlier PR:
https://github.com/lowRISC/opentitan/commit/1987f83b1b732d718b1d79e283b86d4292865f9b#diff-07875233cbf1931b9341dacff929573d2a8e9c3c3c3c6ca3a14a03280fd0c30aR152

https://github.com/lowRISC/opentitan/blob/f600eccc20c7ed9910eb866cc792b71f647095c3/sw/device/silicon_creator/manuf/lib/BUILD#L151-L157